vendor/modules.txt

```diff
@@ -56,7 +56,7 @@ github.com/bgentry/go-netrc/netrc
 github.com/bgentry/speakeasy
 # github.com/davecgh/go-spew v1.1.1
 github.com/davecgh/go-spew/spew
-# github.com/dome9/dome9-sdk-go v1.18.1 => ../dome9-sdk-go
+# github.com/dome9/dome9-sdk-go v1.18.3
```

Comment:
I think this was here for local debugging, (commented out)?
maybe just replace to 1.18.3 and add comment that this is how to use locally?

Reply:
This file is being modified automatically when I use the "go mod vendor" command.
I don't have the ability to change it manually:
![image](https://github.com/dome9/terraform-provider-dome9/assets/138366125/0619f64d-3d81-4d95-91f6-6f8190a59a27)

I think that this is the right way it should be, according to previous commits examples:
https://github.com/dome9/terraform-provider-dome9/commit/3a0d3782f65a4d1a8d63b06da8c4388c3ce98af3
https://github.com/dome9/terraform-provider-dome9/commit/1b175268de60665d47c80778f0dc8a5b7405f58d